src/main/kotlin/gg/essential/vigilance/data/Categories.kt

```diff
@@ -57,6 +57,15 @@ class PropertyItem(val data: PropertyData, val subcategory: String) : CategoryIt
                 protected = false
             )
             PropertyType.BUTTON -> ButtonComponent(data.attributesExt.placeholder, data)
+            PropertyType.CUSTOM -> {
+                val propertyInfoClass = data.attributesExt.customPropertyInfo
+                (propertyInfoClass
+                    .constructors
+                    .find { it.parameters.isEmpty() }
+                    ?.newInstance() as? PropertyInfo)
+                    ?.createSettingComponent(data.getValue())
+                    ?: error("PropertyInfo class $propertyInfoClass has no zero-arg constructor!")
```

Comment:
```suggestion
                propertyInfoClass
                    .getConstructor()
                    .newInstance()
                    .createSettingComponent(data.getValue())
```
Assuming the error thrown by `getConstructor(args...)` is at least as useful as the one we construct manually.